config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
 root 'users#index'
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :new, :create, :show]
+  end
+
+  resources :posts do
```

Comment:
- [ ] Let's remove duplicate code here for the routes

Reply:
resolved